lib/importproject.h

```diff
@@ -102,7 +102,7 @@ class CPPCHECKLIB ImportProject {
 
     void ignorePaths(const std::vector<std::string> &ipaths);
     void ignoreOtherConfigs(const std::string &cfg);
-    void ignoreOtherPlatforms(cppcheck::Platform::PlatformType platformType);
```

Comment:
The last usage was removed in 4745d844ad50ea57f35053971564b2440b9375c8 - looks like a leftover we can simply delete.